packit/api.py

```diff
@@ -292,3 +293,80 @@ def create_srpm(self, output_file: str = None) -> Path:
                 up.bump_spec(version=version, changelog_entry="Development snapshot")
         srpm_path = up.create_srpm(srpm_path=output_file)
         return srpm_path
+
+    def status(self):
+
+        up = Upstream(config=self.config, package_config=self.package_config)
+        dg = DistGit(config=self.config, package_config=self.package_config)
+
+        pr_list = dg.local_project.git_project.get_pr_list()
+        if len(pr_list) > 0:
+            pr_list = pr_list[:3] if len(pr_list) > 3 else pr_list  # take last 3 PRs
+            logger.info("Downstream PRs:")
+            table = [[pr.id, pr.title, pr.url] for pr in pr_list]
+            logger.info(tabulate(table, headers=["ID", "Title", "URL"]))
+        else:
+            logger.info("Downstream PRs: No open PRs.")
+
+        logger.info("\nDist-git versions:")
+        branches = dg.local_project.git_project.get_branches()
+        for branch in branches:
+            try:
+                dg.checkout_branch(branch)
+                logger.info(f"{branch}: {dg.specfile.get_version()}")
```

Comment:
we should also print the branch name and probably say: can't figure out the version instead of 'branch doesnt exist'